contributing/code/bugs.rst

```diff
@@ -34,8 +34,7 @@ If your problem definitely looks like a bug, report it using the official bug
 
 * *(optional)* Attach a :doc:`patch <patches>`.
 
-.. _users mailing-list: http://groups.google.com/group/symfony2
-.. _forum: http://forum.symfony-project.org/
-.. _IRC channel: irc://irc.freenode.net/symfony
+.. _`Stack Overflow`: http://stackoverflow.com/questions/tagged/symfony2
+.. _IRC channel: http://symfony.com/irc
```

Comment:
The switch was done in #5467 after this PR was merged and also updated this file.